src/App.jsx

```diff
@@ -1,19 +1,19 @@
 import React from "react";
 
-//IMPORT STYLES TEST
-import { GlobalStyles, Test } from "./App.styles";
+//IMPORT STYLES 
+import { GlobalStyles } from "./App.styles";
+import SplashPage from "./components/pages/SplashPage";
+//TODO: IMPORT COMPONENTS
+import Footer from "./components/layout/footer/Footer";
+
+//TODO: IMPORT RESET CSS
```

Comment:
The reset CSS should be within the content of the GlobalStyle so no need to have a todo here.